src/main/java/net/mcreator/ui/minecraft/jigsaw/JJigsawPool.java

```diff
@@ -70,11 +71,16 @@ public JJigsawPool(JJigsawPoolsList jigsawPools, IHelpContext gui, JPanel parent
 						true).setAllowEmpty(true));
 		fallbackPool.addPopupMenuListener(new ComboBoxFullWidthPopup() {
 			@Override public void popupMenuWillBecomeVisible(PopupMenuEvent e) {
-				super.popupMenuWillBecomeVisible(e);
-				ComboBoxUtil.updateComboBoxContents(fallbackPool, poolList.stream()
-								.map(p -> jigsawPools.getMCreator().getWorkspace().getWorkspaceSettings().getModID() + ":"
-										+ jigsawPools.getModElement().getRegistryName() + "_" + p.poolName.getText()).toList(),
+				ComboBoxUtil.updateComboBoxContents(fallbackPool,
+						poolList.stream().filter(p -> !p.poolName.getText().isBlank())
+								.filter(p -> p.poolName.getValidationStatus().getValidationResultType()
+										== Validator.ValidationResultType.PASSED)
+								.filter(p -> !p.poolName.getText().equals(poolName.getText()))

```

Comment:
This one is a bit useless as one may want to use the same pool for the fallback (and I even imagine such structures).

Reply:
If this is possible, the help tip should explain what happens in such cases as a fallback to itself may be counter-intuitive without some explanation. Feel free to revert, but also update the help tip to cover this. I can not deduct this information from the current tip:

![image](https://github.com/MCreator/MCreator/assets/16374228/4a677d24-1585-4733-8009-83ebe95de550)